third_party/libev/ev.h

```diff
@@ -696,16 +696,21 @@ EV_API_DECL void ev_resume  (EV_P) EV_NOEXCEPT;
 
 /* these may evaluate ev multiple times, and the other arguments at most once */
 /* either use ev_init + ev_TYPE_set, or the ev_TYPE_init macro, below, to first initialise a watcher */
-#define ev_init(ev,cb_) do {			\
-  ((ev_watcher *)(void *)(ev))->active  =	\
-  ((ev_watcher *)(void *)(ev))->pending = 0;	\
-  ev_set_priority ((ev), 0);			\
-  ev_set_cb ((ev), cb_);			\
+#define ev_init(ev,cb_) do { 							\
```

Comment:
Not sure whether the changes to libev should be placed in a separate commit or not.

Reply:
Discussed f2f: compiler warnings from libunwind sources should be ignored.